symplyphysics/laws/kinematic/constant_acceleration_movement_is_parabolic.py

```diff
@@ -0,0 +1,47 @@
+from symplyphysics import (
+    symbols, Function, Eq, pretty, dsolve, solve, units, Quantity, validate_input, validate_output, expr_to_quantity, simplify
+)
+
+from symplyphysics.definitions import velocity_is_movement_derivative as velocity_definition
+from symplyphysics.definitions import acceleration_is_velocity_derivative as acceleration_definition
+
+# Description
+## Accelerated movement is the kind of movement when object has constant acceleration (e.g with the constant force applied to object).
+## The movement is S = V0*t + a * t**2/2, where
+## S is distance in the moment of time t,
+## V0 is the velocity in the moment of time 0 (initial velocity),
+## a is acceleration,
+## and t is time.
```

Comment:
Add Conditions section and add information that initial coordinate or distance is zero.